nndownload/nndownload.py

```diff
@@ -110,6 +114,16 @@
 BACKOFF_FACTOR = 2  # retry_timeout_s = BACKOFF_FACTOR * (2 ** ({RETRY_ATTEMPTS} - 1))
 TEMP_PATH_LEN = 16
 
+COMMENTS_DATA_JSON = {
+    "globalComments": [
+        {
```

Comment:
No need for this to be an array. It can just be an object: `"globalComments": {...}`